mmsolitaire/mms.html

```diff
@@ -605,39 +605,39 @@
 
 <P>Class variables are not part of the primitive language.  A class
 variable <IMG SRC='_ca.gif' WIDTH=11 HEIGHT=19 ALT='A'> can be
-eliminated from any theorem in which it occurs by replacing it with <IMG
-SRC='lbrace.gif' WIDTH=6 HEIGHT=19 ALT='{'><IMG SRC='_x.gif' WIDTH=10
-HEIGHT=19 ALT='x'> <IMG SRC='vert.gif' WIDTH=3 HEIGHT=19 ALT='|'> <IMG
-SRC='_cpblue.gif' WIDTH=12 HEIGHT=19 ALT='P'><IMG SRC='rbrace.gif'
+eliminated from any theorem in which it occurs by replacing it with
+<IMGRC='lbrace.gif' WIDTH=6 HEIGHT=19 ALT='{'><IMG SRC='_x.gif' WIDTH=10
```

Comment:
Don't know if it is helpful to list all these individually, but in case it is IMGRC should be IMG SRC